seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/psql/PostgresJdbcRowConverter.java

```diff
@@ -189,6 +190,11 @@ public PreparedStatement toExternal(
                             inetObject.setType(PG_INET);
                             inetObject.setValue(String.valueOf(row.getField(fieldIndex)));
                             statement.setObject(statementIndex, inetObject);
+                        } else if (INTERVAL.equalsIgnoreCase(sourceType)) {
+                            PGobject inetObject = new PGobject();
+                            inetObject.setType(INTERVAL);
+                            inetObject.setValue(String.valueOf(row.getField(fieldIndex)));
+                            statement.setObject(statementIndex, inetObject);
```

Comment:
This part didn't cover by test case. Please update postgres e2e test case.

Reply:
ok i will cover it